database/R/R.A. The Rugged Man/All My Heroes Are Dead/Who Do We Trust

```diff
@@ -0,0 +1,96 @@
+Like it was antidisestablishmentarianism
+Until the seeds of despotism arrested my vision
+Lyricism with cynicism and syllogisms
+Until they Pedro Albizu Campos and kill me in prison
+Chemtrail conditions, stem cells of Leo Strauss’ philosophy
+The birth of neo-con policy, but I laughed at America’s fear
+Of a New World Order controllin’ the hemisphere
+’Cause my people been livin’ that for the past 500 years
+
+Intelligence, understanding
+The facts they deserve to know
+No governmental plan
+Should impose its restraints against your will
+Who do we trust? Who do we trust?
+The-the facts-
+Who do we trust? who do we trust?
+Intelligence
+Who can I trust? who can I trust?
+Knowledge of himself-self-self-self-self-
+Who can I trust? Who can I trust?
+
+Expose crimes of the power structure, enter the fire
+’Cause the truth is now treason to the U.S. empire
+U.S meddlin’ in almost every foreign election
+See where the next drone headin’? Civilian dead in the Yemen
+Weddin’ barely reported because the weapons were exported from the politicians in the Pentagon, the Saudis bought it
+And the babies and the children were slaughtered
+The authorization, altercation, deplorable, demoralization
+Obedience to authority, colonize the minority
+Social norm is to conform to the immoral majority
+White superiority, secession of separatism
+The terrorism of American exceptionalism
+Go to plan B, be free, fuck the TV
+Fuck the PD, fuck a Tuskegee VD
+Grave plaque, soldier pop, corpse rot, spin over
+Monsanto weed killer, non-Hodgkin lymphoma
+Whistleblower: men are afraid of, label the threat a traitor
+Executive legislator tryna suppress your behavior
+Intelligence interrogator collectin’ metadata
+Condemnin’ the messenger and censoring the educator
+CIA feed the propaganda to the television
+In the next edition, the pundits preach it in repetition
+Loyal to the monarchy, hereditary or elected
+Fuck a royal family and fuck a president selected, yuh
+
+Secret society
+The secret oath
+Censorship
+I-I-Imposing restraints against your will
+Who do we trust? Who do we trust?
+The-the facts-
+Who do we trust? Who do we trust?
+Intelligence
+Who can I trust? Who can I trust?
+Knowledge of himself-self-self-self-self-self-
+Who can I trust? Who can I trust?
+Informing and alerting
+The American people
+
+What’s the cadet motto at Westpoint?
+“You will not lie, cheat, or steal, or tolerate those who do!”
+I- I- I was a CIA director, we lied, cheated, we stealed- we stole!
+It’s like, we had- we had entire training courses!
+It, It uh- It it it reminds you of the glory of The American Experiment
+
+__________________________________
+Language          American English
+Name              Who Do We Trust?
+Artist            R.A. The Rugged Man, Immortal Technique
+Album             All My Heroes Are Dead
+Track no          19
+Year              2020
+MusicBrainz ID    efc73afe-2ad5-49de-ab20-3a71e334a9d4
+Original text by  Felipe Andres Coronel, R.A. Thorburn, Tone Spliff
```

Comment:
I believe these `, ` need to be replaced by `,  `, too.

Reply:
It's hard to make a unit test for this, since the artist name may differ from directory name due to filesystem limitations, and some artist names contain commas in them.

Reply:
It makes sense, thanks for the explanation 👍